cookbook/integration_crew_ai.ipynb

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
<p>duplication of heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/546/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>